src/lambda/repo.js

```diff
@@ -0,0 +1,34 @@
+import fetch from 'node-fetch';
+
+exports.handler = async function(event, context, callback) {
+	const result = await repoInfo(
+		event.queryStringParameters.repo || 'preactjs/preact'
```

Comment:
same as [other suggestion](https://github.com/preactjs/preact-www/pull/629/files#diff-78d154ba8815cff0a3e10a5ca1f726bdR5).

Reply:
We can't do this here since we allow the user to enter anything here

Reply:
Whoops good point haha